admin/base/urls.py

```diff
@@ -22,10 +23,19 @@
                          url(r'^users/',
                              include('admin.users.urls', namespace='users')),
                          url(r'^prereg/', include('admin.pre_reg.urls', namespace='pre_reg')),
-                         url(r'^reset/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
+                         url(r'^accounts/password_reset/$', 'django.contrib.auth.views.password_reset', name='reset_password'),
+                         url(r'^accounts/password_reset/confirm/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
                             views.password_reset_confirm_custom, name='password_reset_confirm'),
-                         url(r'^reset/done/$', views.password_reset_done,
+                         url(r'^accounts/password_reset/done/$', 'django.contrib.auth.views.password_reset_done', name='password_reset_done'),
+                         url(r'^accounts/password_reset/complete/$',
+                            'django.contrib.auth.views.password_reset_complete',
```

Comment:
Golly. What might be the difference between `password_reset_done` and `password_reset_complete`?